library/src/scripts/dataLists/dataList.tsx

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
The capitalization of this file seems incorrect (lower cased). For case sensitive renames in git try this https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git

Reply:
Good tip, thanks!